src/TemplateBinding.js

```diff
@@ -99,6 +99,53 @@
     };
   }
 
+  // Use for template repeating objects
+  function ObjectArrayProxy(values, observe, onAddedCallback, onRemovedCallback) {
```

Comment:
Maybe ObjectKeysObserver? ("proxy" means something specific in ES6 ... "ArrayProxy" made me think it was a proxy for an array ... whereas the object's keys is established terminology, e.g. [Object.keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys))